Sources/SwiftDocC/Model/Rendering/RenderNodeTranslator.swift

```diff
@@ -779,13 +779,39 @@ public struct RenderNodeTranslator: SemanticVisitor {
             
             return seeAlsoSections
         } ?? .init(defaultValue: [])
+
+        if let callToAction = article.metadata?.callToAction {
+            if let url = callToAction.url {
+                let downloadIdentifier = RenderReferenceIdentifier(url.description)
+                node.sampleDownload = .init(
+                    action: .reference(
+                        identifier: downloadIdentifier,
+                        isActive: true,
+                        overridingTitle: callToAction.buttonLabel(),
+                        overridingTitleInlineContent: nil))
+                downloadReferences[url.description] = DownloadReference(
+                    identifier: downloadIdentifier,
+                    renderURL: url,
+                    sha512Checksum: ""
```

Comment:
Should we make this `nil` instead of an empty string?

Reply:
Currently the `DownloadReference` type (and the corresponding Render JSON spec) requires that `sha512Checksum` is non-nullable. We could add that (i remember a conversation with @mportiz08 saying that we likely should) but i haven't done that yet.

Reply:
I suspect DocC-Render will support a null value already. (Maybe @mportiz08 can confirm?) If so – I'd rather make the spec change as part of this PR since providing an empty string is a little confusing.

Reply:
DocC-Render does not use this field today, so it should be safe to make it optional.

Reply:
@QuietMisdreavus what do you think? Personally I'd rather we just do that here.

Reply:
I think it works to have it as part of this PR. I have this implemented locally already, i'll push it up as soon as i can get it working on symbol pages.